collect_app/src/main/java/org/odk/collect/android/activities/InstanceUploaderList.java

```diff
@@ -111,7 +111,10 @@ public void onClick(View v) {
                     if (checkedItemCount > 0) {
                         // items selected
                         uploadSelectedFiles();
-                        InstanceUploaderList.this.listView.clearChoices();
+                        for (int i = 0; i < listView.getCount(); i++) {
```

Comment:
You can use `setAllToCheckedState`.

Reply:
Good point! thanks, I missed that method.